docs/deploy.md

```diff
@@ -135,3 +135,51 @@ frontend:
 | /<*>.md        | /<*>.md        | 200 (Rewrite) |
 | /<*>.png       | /<*>.png       | 200 (Rewrite) |
 | /<*>           | /index.html    | 200 (Rewrite) |        
+
+
+## Docker
+
+>There is a simple demo for building docsify in docker.
+
+?> tips: you can create all the files for this demo from [Quickstart](https://docsify.js.org/#/quickstart?id=initialize).
```

Comment:
The link to the Quickstart should be provided under the docsify bullet below, not above it. Otherwise, this tip implies "all of the files for this demo" including a docker file, which the Quickstart section does not discuss. Recommend removing this and just adding a link (not a tip) under the docsify bullet below.